homeassistant/components/portainer/quality_scale.yaml

```diff
@@ -53,23 +50,38 @@ rules:
     status: exempt
     comment: |
       No discovery is implemented, since it's software based.
-  docs-data-update: todo
-  docs-examples: todo
-  docs-known-limitations: todo
-  docs-supported-devices: todo
-  docs-supported-functions: todo
-  docs-troubleshooting: todo
-  docs-use-cases: todo
-  dynamic-devices: todo
-  entity-category: todo
-  entity-device-class: todo
-  entity-disabled-by-default: todo
-  entity-translations: todo
-  exception-translations: todo
-  icon-translations: todo
+  docs-data-update: done
+  docs-examples:
+    status: exempt
+    comment: |
+      No examples are there to give.
+  docs-known-limitations:
+    status: exempt
+    comment: |
+      No known limitations at this time.
```

Comment:
Let's document this explicitly. This also invites users to update it if they think that there is one